config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
That is the only one that I am *maybe* going to change to `CAKE_DEBUG`. Though honestly I can't think of another piece of software you would use in conjunction with cake where that is an issue.